apple/internal/macos_binary_support.bzl

```diff
@@ -78,7 +78,6 @@ def _macos_binary_infoplist_impl(ctx):
     actions = ctx.actions
     bundle_name, bundle_extension = bundling_support.bundle_full_name_from_rule_ctx(ctx)
     bundle_id = ctx.attr.bundle_id
-    executable_name = bundling_support.executable_name(ctx)
```

Comment:
this rule didn't have the executable_name attr, but this function was forgiving about that